hash/sega_beena_cart.xml

```diff
@@ -6,10 +6,63 @@ license:CC0
+				<rom name="0 - 0008.png" size="0x1756da2" crc="720920cb" sha1="9291de9a4ecbe42ad3eb0fceb03edaf3824de3e9"  />
+				<rom name="0 - 0009.png" size="0x178b5cb" crc="c1c51104" sha1="a2ff91851ebae16649a06474408bea1a2fcb9aba"  />
+				<rom name="0 - 0010.png" size="0x19850b0" crc="520272eb" sha1="5d40da3e50581201c87f5460bbef1c328675a3cc"  />
+				<rom name="0 - 0011.png" size="0x19b8863" crc="77aa6918" sha1="b2095780c92fb55926e8e26759ebc803fe78d677"  />
+				<rom name="0 - 0012.png" size="0x14630ba" crc="d4a4f360" sha1="af125bddb75aba6ffca1a68dd6f226a15cbd7ce6"  />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="anpawaku" supported="no">
+		<description>Anpanman no Waku Waku Game Oekaki</description>
+		<year>2007</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="アンパンマンのわくわくゲームおえかき"/>
+		<part name="cart" interface="sega_beena_cart">
+			<dataarea name="rom" size="0x800000" width="32" endianness="big">
+				<rom loadflag="load16_word_swap" name="S-100031-1016 - Anpanman no Waku Waku Game Oekaki (Japan).bin" size="0x800000" crc="6330409d" sha1="53d85b60cea717dba39264c9530858fbc621237a" />
+			</dataarea>
+			<dataarea name="pages" size="0x2500000"> <!-- book pages -->
+				<rom name="0 - 0001.png" size="0x1c3a89c" crc="3b8f9a7b" sha1="5b3516229015d264bd4efad905d6c9dc7077a8c9"  />
+				<rom name="0 - 0002.png" size="0x1e83913" crc="bf00ec34" sha1="f0f6454194c2a1c1fba9deca0932e7ca1437b18f"  />
+				<rom name="0 - 0003.png" size="0x2049aa0" crc="ea78bec6" sha1="61a9218ff9b1f4605a47c570f065c1c0dfb88518"  />
+				<rom name="0 - 0004.png" size="0x1eab879" crc="96ca5bc4" sha1="b9ba414d5eaad753b3f8c117f0a08b7170b3f350"  />
+				<rom name="0 - 0005.png" size="0x2118de6" crc="3985b6d4" sha1="c8007d56430b1052b8e78862bfd32ca29b7abc82"  />
+				<rom name="0 - 0006.png" size="0x1efa1a2" crc="5e1bad7c" sha1="43729477aa2fcfa3ef0bdc18c616561105b7c583"  />
+				<rom name="0 - 0007.png" size="0x2182362" crc="769f0a9b" sha1="b4eea9b8c1fdc4c90585aaa57a0d26d28c70aceb"  />
+				<rom name="0 - 0008.png" size="0x1f2078f" crc="abaa43bc" sha1="aa18af322c11830609e22eb1211628514d83304d"  />
+				<rom name="0 - 0009.png" size="0x203043c" crc="84c9bdaa" sha1="ce889774d56ecf2a53acbb60a16b37c01d495efa"  />
+				<rom name="0 - 0010.png" size="0x1c7da38" crc="2c6dfdeb" sha1="4bae2d8ac7dc6edfc559ff467599279f95741c5d"  />
+				<rom name="0 - 0011.png" size="0x1acdaca" crc="9ff9e13e" sha1="2378dd2da016cccd2340668eee9ca1d0540c816a"  />
+				<rom name="0 - 0012.png" size="0x1b3ed31" crc="039b0e84" sha1="f90082cea75ec02618deca28bfbae8ec30e2f023"  />
+			</dataarea>
+		</part>
+	</software>
+
 	<software name="asngnk" supported="no">
 		<description>Shoku Iku Series 1 Soreike! Anpanman-Sukikirai Nai Ko Genki na Ko!</description>
 		<year>2005</year>
 		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="食育シリーズ 1 それいけ！アンパンマン すききらいないこ げんきなこ！"/>
```

Comment:
Please remove the spurious spaces from this.

Reply:
can you please be more specific about the spaces?

Reply:
Japanese is written without spaces.  If different parts of the title need to be clearly separated, use appropriate punctuation.  Also, you should definitely be using a fullwidth “1”, not a halfwidth “1” with spaces around it.

Reply:
Replaced all halfwidth with regular chars and removed spaces!